docs/swagger_v3/contracts.spec.yaml

```
@@ -22,12 +43,15 @@ schemas:
         $ref: '#/components/schemas/TransactionHash'
       source_tx_type:
         type: string
+        $ref: '#/components/schemas/ContractSourceTxType'
         example: ContractCreateTx
```

```suggestion
        $ref: '#/components/schemas/ContractSourceTxType'
```
alternatively, you can inline `ContractSourceTxType` since it is used once